lms/djangoapps/lms_xblock/test/test_runtime.py

```diff
@@ -120,13 +121,13 @@ def mock_get_real_user(_anon_id):
 
         self.runtime = LmsModuleSystem(
             static_url='/static',
-            track_function=Mock(),
-            get_module=Mock(),
-            render_template=Mock(),
+            track_function=Mock(name="track_function"),
+            get_module=Mock(name="get_module"),
+            render_template=Mock(name="render_template"),
             replace_urls=str,
             course_id=self.course_id,
             get_real_user=mock_get_real_user,
-            descriptor_runtime=Mock(),
+            descriptor_runtime=Mock(spec=DescriptorSystem, name="descriptor_runtime"),
```

Comment:
Why is this called "descriptor_runtime" when just above a similar thing was called "descriptor_system"?